app/views/widgets/checkanswer.html

```diff
@@ -1,35 +1,33 @@
-{% macro checkanswer(items, url, common, alreadyDeclared, answerIsCollection) %}
+{% macro checkanswer(items, url, common, alreadyDeclared, answerIsCollection, answerIsGroup) %}
 
 {%  set anyVisibleItems = items | selectattr("visible") | length %}
 {% if anyVisibleItems %}
-<tr>
-    <td colspan="2">
+
     {% set complete = false  %}
     {% for item in items %}
     {% if item.visible %}
     {% if item.answer%}{% set complete = true  %}{% endif %}
-        <div class="grid-row">
-            <div class="column-one-half">{{item.question | safe}}</div>
+        <div class="check-your-answers__row {{'answer-group' if answerIsGroup}}">
```

Comment:
Do we still need `answer-group` as we're now displaying a change link for each line?

Reply:
Done.  Removed answer-group as no longer needed.